ui/client/components/TogglePanel.js

```diff
@@ -1,24 +1,26 @@
 import cn from "classnames"
 import PropTypes from "prop-types"
 import React from "react"
+import {useTranslation} from "react-i18next"
 import "../stylesheets/togglePanel.styl"
 
 import SvgDiv from "./SvgDiv"
 
-export default class TogglePanel extends React.Component {
-
-  static propTypes = {
-    type: PropTypes.oneOf(["RIGHT", "LEFT"]).isRequired,
-    isOpened: PropTypes.bool.isRequired,
-    onToggle: PropTypes.func.isRequired,
-  }
+export default function TogglePanel(props) {
+  const {t} = useTranslation()
+  const {isOpened, onToggle, type} = props
+  const left = type === "LEFT" ? isOpened : !isOpened
+  const iconFile = `arrows/arrow-${left ? "left" : "right"}.svg`
+  const title = type === "LEFT" ?
```

Comment:
Let's do this `LEFT` as enum.

Reply:
enum here gives nothing, only two well known values. string literal types works.